examples/config.sample

```diff
@@ -0,0 +1,115 @@
+# mavlink-router configuration file is composed of sections,
+# each section have some keys and values. They
+# are case insensitive, so `Key=Value` is the same as `key=value`.
+#
+# [This-is-a-section name-of-section]
+# ThisIsAKey=ThisIsAValuye
+#
+# Following specifications of expected sessions and key/values.
+#
+# Section [General]: This section doesn't have a name.
+#
+# Keys:
+#   TcpServerPort
+#       A numeric value defining in which port mavlink-router will
+#       listen for TCP connections. A zero value disables TCP listening.
+#       Default: 5760
+#
+#   ReportStats
+#       Boolean value <true> or <false> case insensitive, or <0> or <1>
+#       defining if traffic statistics should be reported.
+#       Default: false
+#
+#   MavlinkDialect
+#       One of <common> or <ardupilotmega>. Defines which MAVLink dialect
+#       is being used by flight stack, so mavlink-router can log
+#       appropiately.
+#       Default: common
+#
+#   Log
+#       Path to directory where to store flightstack log.
+#       No default value. If absent, no flightstack log will be stored.
+#
+#   LogDebugLevel
+#       One of <error>, <warning>, <info> or <debug>. Which debug log
+#       level is being used by mavlink-router, with <debug> being the
+#       most verbose.
+#       Default:<info>
+#
+# Section [UartEndpoint]: This section must have a name
+#
+# Keys:
+#   Device
+#       Path to UART device, like `/dev/ttyS0`
+#       No default value. Must be defined.
+#
+#   Baud
+#       Numeric value stating baudrate of UART device
+#       Default: 115200
+#
+# Section [UdpEndpoint]: This section must have a name
```

Comment:
I'm just trying to make it clearer - What about something like:
Section UdpEndpoint: This section must have a name - [UdpEndpoint mandatory-section-name]